src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,21 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+
+    @OneToOne
     private Country country;
 
     public Actor() {
```

Comment:
Don't create a default constructor when it is not needed. Since you have already provided a constructor with parameters, the default constructor is not required unless explicitly used for Hibernate or other frameworks that need it.